SCICompanionLib/SCICompanionLib.rc

```diff
@@ -3112,6 +3112,7 @@ END
 IDD_NEWGAMEDIALOG DLGINIT
 BEGIN
     IDC_COMBOLANGUAGE, 0x403, 8, 0, "DOS 437\0",
```

Comment:
At this point, these two lines and the other pair later on should be renamed "DOS US" and "DOS Europe" or something like that.